apps/web-giddh/src/app/reports/components/report-details-components/report.details.component.ts

```diff
@@ -428,6 +428,7 @@ export class ReportsDetailsComponent implements OnInit, OnDestroy {
         if(event) {
             this.monthNames = [this.commonLocaleData.app_months_full.january, this.commonLocaleData.app_months_full.february, this.commonLocaleData.app_months_full.march, this.commonLocaleData.app_months_full.april, this.commonLocaleData.app_months_full.may, this.commonLocaleData.app_months_full.june, this.commonLocaleData.app_months_full.july, this.commonLocaleData.app_months_full.august, this.commonLocaleData.app_months_full.september, this.commonLocaleData.app_months_full.october, this.commonLocaleData.app_months_full.november, this.commonLocaleData.app_months_full.december];
             this.setCurrentFY();
+            this.getSelectedDuration();
```

Comment:
This method returns a string which is unused

Reply:
@prateek-giddh this is being used in html file.